README.md

```diff
@@ -99,3 +99,18 @@ connection.stop([&stop_task](std::exception_ptr exception) {
 
 stop_task.get_future().get();
 ```
+
+### Minimal CMake file
+
+```
+cmake_minimum_required (VERSION 3.5)
+project (signalrclient-sample)
+
+find_path(SIGNALR_INCLUDE_DIR signalrclient/hub_connection.h)
+include_directories(${SIGNALR_INCLUDE_DIR})
+
+find_library(SIGNALR_LIBRARY NAMES signalrclient PATHS {SIGNALR_INCLUDE_DIR} REQUIRED)
+link_libraries(${SIGNALR_LIBRARY})
+
+add_executable (sample sample.cpp)
+```
```

Comment:
At this point you might as well go all the way and include a minimal `sample.cpp`.

Reply:
I forgot that was there all along. Pretend I didn't write this. 😆